designer/server/views/designer.html

```diff
@@ -10,7 +10,7 @@
 
 {% block content %}
     {% if (phase) %}
-    <div class="govuk-phase-banner" role="banner">
+    <div class="govuk-phase-banner--solid" role="banner">
```

Comment:
for BEM, you typically have both the original style `govuk-phase-banner` and then you add the modified class.`class="govuk-phase-banner  govuk-phase-banner--solid"`.  `govuk-phase-banner--solid` should only include the declarations that are different to the base class

Reply:
Updated class as requested